dataland-frontend/src/components/resources/landing/MarketingSection.vue

```diff
@@ -83,4 +83,14 @@ export default defineComponent({
   height: 100%;
   border: 0;
 }
+.top-left-blue-border {
+  border-style: solid;
+  border-width: 1px 0 0 1px;
+  border-color: #a1c5d3;
```

Comment:
Can we specify this color as variable somewhere?

Reply:
I moved the entire style section to an scss file and changed the color source to a variable.